api/metadata/annotations.go

```diff
@@ -30,6 +30,8 @@ const (
 	Profile                     = Domain + "/profile"
 	SecondaryPlatformAnnotation = Domain + "/secondary.platform"
 	OperatorIDAnnotation        = Domain + "/operator.id"
+	RestartedAt                 = Domain + "/restartedAt"
+	Checksum                    = "checksum/config"
```

Comment:
Why don't we add the domain here too (sonataflow.org/checksumConfig)? Is this some kind of default annotation key?